plugins/inputs/snmp_trap/snmp_trap.go

```diff
@@ -42,15 +46,19 @@ type SnmpTrap struct {
 	PrivProtocol string `toml:"priv_protocol"`
 	PrivPassword string `toml:"priv_password"`
 
-	acc        telegraf.Accumulator
-	listener   *gosnmp.TrapListener
-	timeFunc   func() time.Time
-	lookupFunc func(string) (mibEntry, error)
-	errCh      chan error
+	acc      telegraf.Accumulator
+	listener *gosnmp.TrapListener
+	timeFunc func() time.Time
+	errCh    chan error
```

Comment:
we are just removing lookupFunc right?
Can we avoid changes in other lines? The reason why I am suggesting this is, we will have to regularly sync from telegraf git. Fewer the changes, easier it would be to sync.